### ISSUE TITLE
AP is significantly lower than other methods,please help me 

### ISSUE THREAD

**Issue by huoyiyang** [WXinlong/SOLO#177]
Thanks for your error report and we appreciate it a lot.

**Checklist**
1. I have searched related issues but cannot get the expected help.
2. The bug has not been fixed in the latest version.

**Describe the bug**
I use my dataset with coco dataet format, other method such as mask rcnn , gc net ,yolact implemented by mmdetection all work well. all on colab!
but when i use in  solov2 1x r50 ,there are several question!
(1)the loss_ins can not decrease , so i try to change lr, the lr can down,but still stop at around 0.3
(2)the test AP is significantly lower than other methods,about 20%
(3)i try to change the hyperparameter grid ,i enlarge them ,it seems useless

**Reproduction**
1. What command or script did you run?
!python tools/train.py configs/solov2/solov2_r50_fpn_8gpu_1x.py

2. Did you make any modifications on the code or config? Did you understand what you have modified?
i change the lr ,with your origin lr, the loss_ins stay=3

3. What dataset did you use?
my dataset ssdd ,no problem
**Environment**
i change your modified code to torch 1.4,torchcheck to atcheck

**Error traceback**
If applicable, paste the error trackback here.
```
![image](https://user-images.githubusercontent.com/73746988/122396688-d3d0d280-cfaa-11eb-89ad-f2c80b174268.png)

A placeholder for trackback.
```

**Bug fix**
If you have already identified the reason, you can provide the information here. If you are willing to create a PR to fix it, please also leave a comment here and that would be much appreciated!
